src/viam/sdk/module/service.cpp

```diff
@@ -114,6 +114,7 @@ ::grpc::Status ModuleService_::ReconfigureResource(
         res->reconfigure(deps, cfg);
         return grpc::Status();
     } catch (const std::exception& exc) {
+        return grpc::Status(::grpc::INTERNAL, exc.what());
```

Comment:
The only way for `reconfigure` calls to fail is to throw an exception, so it is important to handle this case. We should probably add a `catch(...)` here too just in case.